ext/intl/transliterator/transliterator_class.c

```diff
@@ -144,22 +144,12 @@ static zend_object *Transliterator_clone_obj( zend_object *object )
 
 	if( to_orig->utrans != NULL )
 	{
-		zval			tempz; /* dummy zval to pass to transliterator_object_construct */
-
 		/* guaranteed to return NULL if it fails */
 		UTransliterator *utrans = utrans_clone( to_orig->utrans, TRANSLITERATOR_ERROR_CODE_P( to_orig ) );
 
-		if( U_FAILURE( TRANSLITERATOR_ERROR_CODE( to_orig ) ) )
-			goto err;
-
-		ZVAL_OBJ(&tempz, ret_val);
-		transliterator_object_construct( &tempz, utrans,
-			TRANSLITERATOR_ERROR_CODE_P( to_orig ) );
-
 		if( U_FAILURE( TRANSLITERATOR_ERROR_CODE( to_orig ) ) )
 		{
```

Comment:
If we use 1tbs for the `else`, we should also use it here:
```suggestion
		if( U_FAILURE( TRANSLITERATOR_ERROR_CODE( to_orig ) ) ) {
```
I'd also "fix" the strange whitespace in this line, but it appears to be consistent with the rest of the file.

Reply:
alright, updated